hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -130,6 +130,18 @@ function ozonecmd_case
       OZONE_RUN_ARTIFACT_NAME="hadoop-ozone-tools"
     ;;
     genesis)
+      ARTIFACT_LIB_DIR="${HADOOP_HDFS_HOME}/share/ozone/lib/hadoop-ozone-tools"
+      mkdir -p "$ARTIFACT_LIB_DIR"
+      if [[ ! -f "$ARTIFACT_LIB_DIR/jmh-core-1.23.jar" ]]; then
+        echo "jmh-core jar is missing from $ARTIFACT_LIB_DIR, trying to download from maven central (License: GPL + classpath execption)"
+        wget -O "$ARTIFACT_LIB_DIR/jmh-core-1.23.jar"  https://repo1.maven.org/maven2/org/openjdk/jmh/jmh-core/1.23/jmh-core-1.23.jar
```

Comment:
How do we choose between `wget` and `curl`?

Reply:
> How do we choose between wget and curl?

`if random.getNextInt() % 2 == 0 --> wget` 

;-)

Will replace it with `curl` if that one is your preference. Is it more common?

Reply:
> Is it more common?

I don't have numbers for that.

https://daniel.haxx.se/docs/curl-vs-wget.html says:

> More portable. curl builds and runs on lots of more platforms than wget. For example: OS/400, TPF and other more "exotic" platforms that aren't straight-forward unix clones.